hack/netpol-generator/install-cyclonus.yml

```diff
@@ -0,0 +1,21 @@
+apiVersion: batch/v1
+kind: Job
+metadata:
+  name: cyclonus
+  namespace: kube-system
+spec:
+  template:
+    spec:
+      restartPolicy: Never
+      containers:
+      - command:
+        - ./cyclonus
+        - generate
+        - --mode=upstream
+        - --noisy=true
+        - --perturbation-wait-seconds=15
+        - --cleanup-namespaces=true
+        name: cyclonus
+        imagePullPolicy: Always
```

Comment:
an `imagePullPolicy` of `Always` doesn't really make sense if you used a versioned image?

Reply:
ah good call 👍